units/other/dummycom.lua

```diff
@@ -0,0 +1,50 @@
+return {
+	dicecom = {
```

Comment:
```suggestion
	-- implemented as an actual unit partially because stuff assumes choice is always a real unit
	-- and partially because the dice model is still drawn pregame so allies get visual feedback "for free"
	dicecom = {
```